tests/system/src/test/java/test/javafx/scene/web/StraightLineTest.java

```diff
@@ -132,48 +135,47 @@ public void setupTestObjects() {
                     "<html>\n" +
                     "<head>\n" +
                     "<style>\n" +
+                    "body {\n" +
+                    "margin:0px;\n"+
+                    "}\n" +
                     "div {\n" +
                     "padding:0px;\n"+
-                    "  text-decoration: underline;\n" +
-                    "  text-decoration-thickness: 20px;\n" +
+                    "width:150px;\n"+
+                    "height:20px;\n"+
+                    "margin:0px;\n"+
+                    "text-decoration: underline;\n" +
+                    "text-decoration-thickness: 30px;\n" +
                     "font-size: 16px; \n" +
                     "}\n" +
                     "</style>\n" +
                     "</head>\n" +
                     "<body>\n" +
-                    "<div>TEST</dv>\n" +
+                    "<div>TEST UNDERLINE</div>\n" +
                     "</body>\n" +
                     "</html>");
-
         });
 
         assertTrue("Timeout when waiting for focus change ", Util.await(webViewStateLatch));
-        Util.sleep(1000);
 
         Util.runAndWait(() -> {
             WritableImage snapshot = straightLineTestApp.primaryStage.getScene().snapshot(null);
             PixelReader pr = snapshot.getPixelReader();
-
             int start_x = (int)webView.getEngine().executeScript("document.getElementsByTagName('div')[0].getBoundingClientRect().x");
             int start_y = (int)webView.getEngine().executeScript("document.getElementsByTagName('div')[0].getBoundingClientRect().y");
             int height = (int)webView.getEngine().executeScript("document.getElementsByTagName('div')[0].getBoundingClientRect().height");
             int width = (int)webView.getEngine().executeScript("document.getElementsByTagName('div')[0].getBoundingClientRect().width");
 
-            // buttom start x position of underline ( startx + font size + thickness -1)
-            int line_start_x = start_x + height + 20 - 1;
-            // buttom start y position of underline ( startx + height)
-            int line_start_y = start_y + height + 3;
-            String expected_line_color = "rgba(0,0,0,255)"; // color of line
-            for (int i = line_start_y; i <= (width - line_start_y -6); i++) {
-                String actual_line_color = colorToString(pr.getColor(line_start_x, i));
-                if (actual_line_color.equals(expected_line_color)) {
-                    continue;
-                }
-                else {
-                    fail("Each pixel color of line should be" + expected_line_color + " but was:" + actual_line_color);
-                }
+            int line_start_x = start_x;
+            int line_end_x = start_x + width - SKIP_TEXT_BOUNDARY;
+            int line_start_y = start_y + height + LINE_THICKNESS/2;
+            String line_color = "rgba(0,0,0,255)"; // color of line
+            System.out.println(line_end_x);
+            System.out.println(line_start_y);
```

Comment:
Once you are done debugging the test, you can remove these print statements.

Reply:
ok,